e2e/tests/modules/esl-media/utils/utils.ts

```diff
@@ -0,0 +1,16 @@
+import type {ESLMedia} from '@exadel/esl/modules/esl-media/core/esl-media';
+import type {JSHandle} from 'puppeteer';
+
+export const createMediaElement = async (props: Partial<ESLMedia>): Promise<JSHandle<ESLMedia>> => {
+
+  return page.evaluateHandle((pr: Partial<ESLMedia>) => {
+    const toKebabCase = (str: string): string => str.replace(/([a-z])([A-Z])/g, '$1-$2').replace(/[\s_]+/g, '-').toLowerCase();
+
+    const $media = document.createElement('esl-media');
+    Object.entries(pr).forEach(([attr, value]) => {
+      $media.setAttribute(toKebabCase(attr), value?.toString() || '');
```

Comment:
Propbaly in case ESLMedia is already registered you can just do

`Object.assign(ESL.Media.create(), props);` or
`Object.assign(document.createElement('esl-media'), props);`